index.html

```diff
@@ -11,22 +11,200 @@
+                            <h2 class="skills">Languages</h2>
+                            <ul class="language">
+                            
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">css</button></li>
+                            <li><button type="button">Java Script</button></li>
+                            <li><button type="button">HTML</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box2">
+                        <div class="skills">
+                            <img src="images/icon-frameworks.svg" alt="icon Frameworks" class="icon">
+                            <h2 class="skills">Frameworks</h2>
+                            
+                            <ul class="language">
+                            <li><button type="button">Bootstrap</button></li>
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">RSpec</button></li>
+                            <li><button type="button">Capybara</button></li>
+                            <li><button type="button">Selenium</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box3">
+                        <div class="skills">
+                            <img src="images/icon-skills.svg" alt="icon skills" class="icon">
+                            <h2 class="skills">Skills</h2>
+                            <ul class="language">
+                            <li><button type="button">Codekit</button></li>
+                            <li><button type="button">GitHub</button></li>
+                            <li><button type="button">Codepen</button></li>
+                            <li><button type="button">Gitlab</button></li>
+                            <li><button type="button">Terminal</button></li>
+                        </ul>
+                        </div>        
+                    </div>
+                </div>
+            <br>
+            <div class="aboutyou">
+                <div class="comment"><h2 class="aboutyou-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+                <form class="form" action="https://formspree.io/f/mbjwboje" method="post">
+                    <input class="textarea" type="text" placeholder="name" maxlength="30" required>
+                    <input class="textarea" type="email" placeholder="email" required>
+                    <textarea class="textarea" name="mdr_message" id="message" cols="50" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                   <button class="projects" type="submit">Get in touch
+                        </button>
+                </form>
+            </div>
```

Comment:
- [ ] Kindly align the footer properly and add the `hr` before the icons. also, the email is not displayed in the desktop view, please try to remove it from the design

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_164630](https://user-images.githubusercontent.com/25227022/172889851-8d2e2667-c845-4bc6-b279-fdebd4764aaf.png)|![Screenshot_20220609_164644](https://user-images.githubusercontent.com/25227022/172889884-705eedc0-cd8f-4754-b74b-78c55770c367.png)|